### PR TITLE
Fix typo in Contributing.md

### LINKED ISSUE #819: Fix typo in Contributing.md
### Description of feature

There is a typo https://ehrapy.readthedocs.io/en/stable/contributing.html in the writing tests section… pip install '-e[dev,test]' should be pip install -e . ‘[dev,test,docs]’, both fixing the -e within the ‘’ and harmonizing the install instruction with the earlier instruction.

### PR DESCRIPTION
**Pull request by aGuyLearning**
Fixes #819

### CODE REVIEW

File: docs/contributing.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ and [prettier][prettier-editors].
 ## Writing tests
 
 ```{note}
-Remember to first install the package with `pip install '-e[dev,test]'`
+Remember to first install the package with `pip install -e "[dev,test,docs]"`
```
Comment (Zethson) [inline docs/contributing.md:54]: Sorry and great catch!

File: tests/plot/test_catplot.py [hunk 2]
```diff
@@ -11,6 +11,6 @@ def test_catplot_vanilla(adata_mini, check_same_image):
 
     check_same_image(
         fig=fig,
-        base_path=f"{_TEST_IMAGE_PATH}/catplot_vanilla",
+        base_path=f"{_TEST_IMAGE_PATH}/catplot_vanilla",;
```
Comment (Zethson) [inline tests/plot/test_catplot.py:14]: Why are you adding a `;` here?

**Review by Zethson — COMMENTED** [review]
Great! Welcome.

File: tests/plot/test_catplot.py (hunk 2 above)
Comment (eroell) [reply tests/plot/test_catplot.py:14]: I asked @aGuyLearning to make something fail to see how the CI flares up :)

**Comment by aGuyLearning** [comment]
Run Notebooks test seems to fail, when changes are not made in ehrapy repo?

**Pull request closed without merging**